src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java

```diff
@@ -190,38 +190,6 @@ public CaseEndpoint(
     return ResponseEntity.ok(caseDetailsDTOList);
   }
 
-  @RequestMapping(method = RequestMethod.GET)
-  public ResponseEntity<List<CaseDetailsDTO>> findCases(
-      @RequestParam(required = false) String sampleUnitId,
-      @RequestParam(required = false) String partyId) {
-    log.with("sample_unit_id", sampleUnitId).with("party_id", partyId).debug("Finding cases");
-    Example<Case> exampleCase = buildExampleCase(sampleUnitId, partyId);
-    List<Case> cases = getCases(exampleCase);
-    List<CaseDetailsDTO> caseResponses =
-        cases
-            .stream()
-            .map(
-                c -> {
-                  CaseDetailsDTO caseDetails = mapperFacade.map(c, CaseDetailsDTO.class);
-                  CaseGroup parentCaseGroup =
-                      caseGroupService.findCaseGroupByCaseGroupPK(c.getCaseGroupFK());
-                  caseDetails.setCaseGroup(mapperFacade.map(parentCaseGroup, CaseGroupDTO.class));
-                  return caseDetails;
-                })
-            .collect(Collectors.toList());
-
-    return ResponseEntity.ok(caseResponses);
-  }
-
-  private List<Case> getCases(Example<Case> exampleCase) {
-    boolean emptyRequest = exampleCase.getProbe().equals(new Case());
-    if (emptyRequest) {
-      return caseRepository.findAll();
-    } else {
-      return caseRepository.findAll(exampleCase);
-    }
-  }
-
   private Example<Case> buildExampleCase(String sampleUnitId, String partyId) {
```

Comment:
This can be deleted too as the only thing that used it is in the deleted block above

Reply:
I thought I'd done that, good spot!